playground/backend/internal/db/schema/migration/migrations_test.go

```diff
@@ -61,8 +57,9 @@ func setup() {
 }
 
 func teardown() {
-	if err := datastoreDb.Client.Close(); err != nil {
-		panic(err)
+	clientCloseErr := datastoreDb.Close()
+	if clientCloseErr != nil {
+		panic(clientCloseErr)
```

Comment:
Perhaps these errors could be collected in chan error and handled like:

```
for {
   select {
      case err := <- clientErr:
           // gracefully shut down other processes
           return err
     case err := <- datastoreDbErr:
           // gracefully shut down other processes
           return err
   }
}
```

Reply:
We don't have any other processes here and I don't think there is much merit in complicating the error handling situation here with channels. Current code, while not exactly following Go conventions to the letter, is cleaner in my opinion.